test/CodeGen/AArch64/pagerando-optimizer.ll

```diff
@@ -0,0 +1,23 @@
+; RUN: llc < %s -march=aarch64 -relocation-model=pip -o - | FileCheck %s
+
+define void @wrapper() pagerando_wrapper { ret void }
+define internal void @orig() pagerando_binned { ret void }
+
+define internal void @user() pagerando_binned {
+  call void @wrapper()
+  call void @orig()
+  ret void
+}
+
+; CHECK-LABEL: .text
```

Comment:
I think we should put the checks inline where the functions are defined (i.e. `CHECK-LABEL: wrapper` right before or after `define void @wrapper()`.

Reply:
Roger that. For some tests this was difficult because our `PagerandoWrappers` appends wrappers at the end. I will do it for all tests where it possible.